src/tools/perf/libperf.c

```diff
@@ -428,6 +439,17 @@ static ucs_status_t uct_perf_test_check_capabilities(ucx_perf_params_t *params,
         }
     }
 
+    if ((UCT_PERF_DATA_LAYOUT_ZCOPY == params->uct.data_layout)) {
+        if (params->uct.iovcnt > max_iov) {
+            if (params->flags & UCX_PERF_TEST_FLAG_VERBOSE) {
+                ucs_error("Number of IOV entries exceeded. Maximum is -i %lu", max_iov);
```

Comment:
pls remove "-i" - it's only a paramter for perftest application, but libperf can be also used w/o it, for example from gtest